src/transformers/models/auto/modeling_tf_auto.py

```diff
@@ -187,6 +188,7 @@
 
 TF_MODEL_FOR_MASKED_IMAGE_MODELING_MAPPING_NAMES = OrderedDict(
     [
+        ("beit", "TFBeitForMaskedImageModeling"),
```

Comment:
See [docstring in PT version](https://github.com/huggingface/transformers/blob/fb366b9a2a94b38171896f6ba9fb9ae8bffd77af/src/transformers/models/beit/modeling_beit.py#L723)

```suggestion
```